ROSNET/ROSNET.csproj

```diff
@@ -2,13 +2,28 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Would suggest to just use .NET 6, (after looking at https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/, I see no reason to use .NET Standard 2.1), and with `<LangVersion>10</LangVersion>`